pages/1.13/release-notes/1.13.0-beta1/index.md

```diff
@@ -0,0 +1,207 @@
+<no content>
+
+- [Kubernetes] Edge-LB TLS/SNI Integration (DCOS-47322)
+<no content>
+
+## GUI
+- Independent Upgrade of DC/OS UI (DCOS-47632)
+
+    Customers will now be able to get and updated version of the DC/OS GUI without having to upgrade DC/OS.  New Updates in DC/OS will be published to the DC/OS catalog and will also be available as .dcos file for our on-premise customers. This will allow our customers to easily get the latest fixes and capabilities in our GUI. Customers will  also be able to roll back to the original GUI version that was shipped with their version of DC/OS if they need to.
+
+- Accurate Service Status (Part 1: Plan Tab) (DCOS-43460)
+
+    DC/OS 1.13 GUI now includes a new tab in the details section of every SDK based services. With this new tab, Operator will have a clear indication of the status and progress of sdk-services during its lifecycle such as installing, upgrade. Customers can have a details about the specific plans running or just finished up to the execution of each task so that they can easily track the progress among those plans
+
+## Installation
+- Multi-Region Support on Universal Installer (DCOS-45727)
+
+    Multi-region deployments enable higher availability for DC/OS clusters and is a crucial item to have for customers that are wanting to achieve uptime without being susceptible for regional outages.
+
+- Dynamic Masters on Universal Installer (DCOS-45725)
+
+    Dynamic masters is the ability to create, destroy and recover master nodes. This enables users on the Universal Installer to be able to downscale or upscale their clusters from not just the agent nodes (which is currently supported), but also from the master nodes as well - if they deem necessary.
+
+- Universal Installer & on-premise DC/OS lifecycle management w/Ansible (DCOS-45724)
+
+    dcos-ansible is a Mesosphere provided and supported ansible roles for installing and upgrading Mesosphere DC/OS and DC/OS Enterprise on any user provided infrastructure.
+
+<!-- not in 1.13 Docs with RN filter 
+- Universal Installer to provision EBS volumes (DCOS-47221)
+The Universal Installer now provides the ability to provision AWS EBS volumes and attach them to the Private Agents within a cluster.
+
+Documented here: https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/install/provision-extra-volumes/
+-->
+
+## Metrics
+- [dcos-telegraf] Instrument and Transmit telegraf Metrics to via telegraf (DCOS-39012)
+
+    dcos-telegraf collects and forwards metrics about itself for storage and reporting via dcos-monitoring or third party monitoring services.
+
+## Networking
+- Easily obtainable Public-IP of the public agents through DC/OS Net API endpoint (DCOS-28127)
+
+## Platform
+- Update Rexray to support NVMe EBS volumes (DCOS-50047)
+
+    In DC/OS 1.13.0, REX-ray support has been updated to include EBS NVMe volumes.
+
+- [UI] Internationalization I18N & L10N (Chinese) (DCOS-39557)
```

Comment:
Why is this not included in the UI section

Reply:
Moved localization info to the GUI section